ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml

```diff
@@ -413,5 +413,95 @@ ViewData["Title"] = "Notification - Clinical Details";
             </div>
         </div>
     </date-comparison>
+    <br/>
+
+    <h2>Treatment Options</h2>
+    <validate-multiple model="ClinicalDetails" :properties="['IsShortCourseTreatment', 'IsMDRTreatment']" inline-template>
```

Comment:
Just to double check, the colon in `:properties` is intentional?

Reply:
yes - it's so that vue interprets this as javascript and maps it correctly to an array; otherwise it will just think it's a string starting with a square bracket

Reply:
ah gotcha - nice trick!